controllers/storagecluster/provider_server.go

```diff
@@ -0,0 +1,272 @@
+package storagecluster
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+
+	"go.uber.org/multierr"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
+)
+
+var (
+	ServerImage = GetEnvVarOrPanic("PROVIDER_API_SERVER_IMAGE")
```

Comment:
```suggestion
	serverImage = GetEnvVarOrPanic("PROVIDER_API_SERVER_IMAGE")
```
Name is not used outside the package.

Reply:
Not now, We may use them in the e2e tests which are not part of the same pkg.